src/Microsoft.ML.Data/DataLoadSave/Binary/Codecs.cs

```diff
@@ -154,27 +154,6 @@ private sealed class UnsafeTypeCodec<T> : SimpleCodec<T> where T : struct
 
             private readonly UnsafeTypeOps<T> _ops;
 
-            public override string LoadName
-            {
-                get
-                {
-                    switch (Type.RawKind)
-                    {
-                        case DataKind.I1:
-                            return typeof(sbyte).Name;
-                        case DataKind.I2:
-                            return typeof(short).Name;
-                        case DataKind.I4:
-                            return typeof(int).Name;
-                        case DataKind.I8:
-                            return typeof(long).Name;
-                        case DataKind.TS:
-                            return typeof(TimeSpan).Name;
-                    }
-                    return base.LoadName;
-                }
-            }
```

Comment:
cc @yaeldekel in case it breaks something in TLC

---
In reply to: [249104896](https://github.com/dotnet/machinelearning/pull/2176#discussion_r249104896) [](ancestors = 249104896)